components/header/_header2.scss

```diff
@@ -0,0 +1,65 @@
+$header2-bp: 36em;
+
+.header2 {
```

Comment:
This should be named something more meaningful than `.header2`. Are we even keeping the old header around? If so, it's probably a modifier of the existing `.header` eg. `.header--new`

Reply:
Have reverted the class name to remove class references with '2' in them. This doesn't appear to be a modifier but a fix of the original header.